docs/index.md

```diff
@@ -0,0 +1,58 @@
+# AWS Account Operator
```

Comment:
Should we point the main README to link to this page instead, so we don't have to update multiple places.